### ISSUE TITLE
XBB.1.5 sublineage with S:S494P and N:N77T/ORF9b:I74L (52 seqs as of 2023-04-18, mainly Singapore)

### ISSUE THREAD

**Issue by c19850727** [cov-lineages/pango-designation#1910]
### Description
**Sublineage of**: XBB.1.5
**Earliest sequence**: 2023/03/06 (Singapore)
**Most recent sequence**: 2023/04/11 (Singapore)
**Countries circulating**: Singapore (51 local cases), Japan (1 local case)

**Mutations on top of XBB.1.5**: C18501T > C8092T, C10782T **(ORF1a:T3506I)**, T23042C **(S:S494P)** > A28503C (**N:N77T/ORF9b:I74L**)

**GISAID and Cov-spectrum query**: **C16391T, C18348A, G26553C**

### Evidence
**Usher tree**:
![image](https://user-images.githubusercontent.com/86835836/232744452-35ea2ecb-0140-4e62-bbf0-46a24314e632.png)
https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_2de3_e681a0.json?c=gt-nuc_8092,28503&label=id:node_5884181

Also there are 5 more sequences on GISAID with S:S494P but without N:N77T/ORF9b:I74L, 2 of which are imported cases caught by US border surveillance with a travel history from Belgium in Feb 2023.

**Cov-Spectrum growth rate**:
In recent 2 months in Singapore, this proposed sublineage has 60% growth advantage relative to XBB.1.5*, 61% relative to XBB.1.9*, and almost on par with XBB.1.16* (-5%).

![image](https://user-images.githubusercontent.com/86835836/232745182-cc325ff9-e49d-4210-acc3-c6b2a71bd751.png)

### Genomes:
EPI_ISL_17207348, EPI_ISL_17207361, EPI_ISL_17207368, EPI_ISL_17262381, EPI_ISL_17262435, EPI_ISL_17346958, EPI_ISL_17346963, EPI_ISL_17346990, EPI_ISL_17346994, EPI_ISL_17347001, EPI_ISL_17347027, EPI_ISL_17347035, EPI_ISL_17347205-17347206, EPI_ISL_17347211, EPI_ISL_17388028, EPI_ISL_17388412, EPI_ISL_17388474, EPI_ISL_17388499, EPI_ISL_17388515, EPI_ISL_17388528, EPI_ISL_17388539, EPI_ISL_17388584, EPI_ISL_17388595, EPI_ISL_17388624, EPI_ISL_17388647, EPI_ISL_17388662, EPI_ISL_17388692, EPI_ISL_17388790, EPI_ISL_17388814, EPI_ISL_17431911-17431912, EPI_ISL_17431914, EPI_ISL_17431929, EPI_ISL_17431997-17431998, EPI_ISL_17432034, EPI_ISL_17432125, EPI_ISL_17432176, EPI_ISL_17432212, EPI_ISL_17432246, EPI_ISL_17496122, EPI_ISL_17496124, EPI_ISL_17496126, EPI_ISL_17496129, EPI_ISL_17496132, EPI_ISL_17496137, EPI_ISL_17496144, EPI_ISL_17496149, EPI_ISL_17496154, EPI_ISL_17496333, EPI_ISL_17496406

**Comment by FedeGueli** [comment]
@c19850727  i think the correct query is : **C18501T , C8092T, C10782T ,T23042C**, yours doent seem to catch this lineage.

**Comment by c19850727** [comment]
Yes Fede thanks for pointing out! it must be a silly copy/paste error. let me edit the proposal now.

**Comment by FedeGueli** [comment]
one sample from Canada today

**Comment by corneliusroemer** [comment]
I'd rather not use growth advantages when there are so few samples. Now it's disappearing:
<img width="1910" alt="image" src="https://user-images.githubusercontent.com/25161793/233853510-ddb55969-195a-4171-8622-1a19c7da21fc.png">
https://cov-spectrum.org/explore/Singapore/AllSamples/Past2M/variants?nextcladePangoLineage=XBB.1.5*&nucMutations1=C18501T%2CT23042C&nextcladePangoLineage1=XBB.1.5*&analysisMode=CompareToBaseline&

It's pretty clear that 478 is the most potent mutation. Other mutations have had time to happen and haven't shown much success comparatively so the prior for a single Spike mutation other than 478 to have big growth advantage is quite small now, that includes 494.

**Comment by corneliusroemer** [comment]
I've designated the parent branch as XBB.1.5.43

**Comment by c19850727** [comment]
65 seqs as of 2023-04-28.

**Comment by c19850727** [comment]
71 seqs as of 2023-05-05. Newly found in China-Shanghai and India-MH.

**Comment by c19850727** [comment]
75 seqs as of 2023-05-12. Newly found in Vietnam.

**Comment by DailyCovidCases** [comment]
Please close this issue

**Comment by DailyCovidCases** [comment]
Please close this issue

**Comment by DailyCovidCases** [comment]
Please close this issue

**Comment by DailyCovidCases** [comment]
This branch is dead, please close this issue
